tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1235,6 +1270,18 @@ When(
   }
 )
 
+When('the user selects following files/folders to copy/move using the webUI', async function(
```

Comment:
We may already have a step for batch selection. If not maybe change this step to 
`the user selects following files/folders for batch action using the webUI`